src/components/endpoints/Mint.vue

```diff
@@ -88,6 +194,63 @@ const mint = async () => {
           </div>
         </div>
       </div>
+      <div v-else>
+        <div class="field">
```

Comment:
Rather then duplicating I would move this form into own component so it can be shared in Asses and Mint